week-2-practise/level-1/5-indexOf.js

```diff
@@ -3,7 +3,7 @@
 
 var itemToFind = 7;
 var arr = [3, 5, 61, 7, 123];
-var index; // ONLY EDIT THIS LINE
+var index = arr.indexOf("h"); // ONLY EDIT THIS LINE
```

Comment:
should be
```suggestion
var index = arr.indexOf(itemToFind); // ONLY EDIT THIS LINE
```

Reply:
Of course,,,,, even there is no h value in the given array,,,, I put it there when I was giving to some examples my class mates,,,,it's my bad 😉